LayoutTests/fast/attachment/mac/attachment-keynote-expected.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html><!-- webkit-test-runner [ AttachmentElementEnabled=true ] -->
+<html class="reftest-wait">
+<body>
+<attachment></attachment>
+<script>
+if (window.internals)
+    file = window.internals.createFile("../resources/test-file.key");
+
+var attachments = document.getElementsByTagName("attachment");
+for (var i = 0; i < attachments.length; i++)
+    attachments[i].file = file;
```

Comment:
Looking at these tests, the empty Keynote/Pages/Numbers, and the fact that the previous files were `-expected-mismatch.html`, my understanding is that this previously tested the fact that attachment elements drew something on the page.

Now, these tests appear to be testing that setting the file programmatically, and with an attribute, have the same result. Is that understanding correct? If so, I think we should preserve the old test behaviors while also adding new tests.

Reply:
I agree with Aditya, it's probably more useful to have an -expected-mismatch.html file that is the exact same attachment but with a different icon.

Reply:
Thanks for the reviews.
Yeah, I think I got carried away by previous patches (that was testing `.txt` icon set two ways), and I didn't quite get the importance of checking that we were drawing *any*thing. 😅
Alright, I'll see if I can make the old mismatch tests stable (and compare with another type of attachment, good idea @nt1m), and I'll add these new ones as well.